website/screens/components/table/specs/TableSpecsPage.tsx

```diff
@@ -0,0 +1,543 @@
+import {
+  DxcTable,
+  DxcStack,
+  DxcText,
+  DxcList,
+  DxcLink,
+} from "@dxc-technology/halstack-react";
+import Image from "@/common/Image";
+import HeadingLink from "../../../common/HeadingLink";
+import anatomyImage from "./images/table_anatomy.png";
+import specsImage from "./images/table_specs.png";
+import Figure from "../../../common/Figure";
+import DocFooter from "../../../common/DocFooter";
+
+const TableSpecsPage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Specifications</HeadingLink>
+        <Figure caption="Table design specifications">
+          <Image src={specsImage} alt="Table design specifications" />
+        </Figure>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>States</HeadingLink>
+        <DxcText as="p">
+          Some other components defined in the design system are used in the
+          table component as the{" "}
+          <DxcLink
+            href="https://developer.dxc.com/design/guidelines/components/checkbox"
+            text="checkbox"
+            newWindow
+          />
+          ,{" "}
+          <DxcLink
+            href="https://developer.dxc.com/design/guidelines/components/button"
+            text="button"
+            newWindow
+          />{" "}
+          or{" "}
+          <DxcLink
+            href="https://developer.dxc.com/design/guidelines/components/select"
+            text="select"
+            newWindow
+          />
+          . For concrete specifications about states, please, consider to see
+          the documentation of each component.
+        </DxcText>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Anatomy</HeadingLink>
+        <Figure caption="Table component anatomy">
```

Comment:
Anatomy Images don't need a `Figure` surrounding them, we should remove it.

Reply:
Done.